CMakeLists.txt

```diff
@@ -54,7 +54,7 @@ set(libtorrent_include_files
 	enum_net
 	error
 	error_code
-	extensions
+	extensions.hpp
```

Comment:
is this right?

Reply:
yes, it's something that have been bothering me for a long time. CLion (and `cmake`) pick the folder `include/extensions` as project element, this way only `extensions.hpp` is picked up.

Reply:
interesting. The purpose of this list of header files is only for creating a package, right?
I just realized there's no logic that adds the ".hpp" suffix to these names. is that implied by cmake?
when building a package with cmake, does it still include the `extensions` directory? I imagine it would need to be included too

Reply:
Yes, `cmake` performs all kind of extensions guessing until it finds (or not) the file. The files under `extensions` are still included with the logic associated to `libtorrent_extensions_include_files`.

Reply:
it seems reasonable then to add the `.hpp` extension to all files here, what do you think?

Reply:
hmm... maybe in the future :)

Reply:
I think it would make sense to do this as a separate patch. there's nothing in this PR that depends on this fix, is there?

Reply:
no, removing this `.hpp` from this PR